### ISSUE TITLE
Specify how metadata is organized (in the context of raw data organization)

### ISSUE THREAD

**Issue by jcolomb** [gwappa/physiology-data-organization#1]
One important thing in bids is that the metadata is to be saved in tsv or json format. This should be indicated.

You have also no metadata file in your proposed structure.

**Comment by gwappa** [comment]
yes, it is because I have not yet finished with how to organize metadata.
it is going to be integrated.

and rDOPE itself will only be about management of raw data. It seems to me
that we cannot fix how the metadata is stored unless we have completely
fixed way of experiments, such as with the use of human MRI.

2019年10月28日(月) 9:59 Julien Colomb <notifications@github.com>:

**Comment by gwappa** [comment]
You can check what I try to implement [here](https://amorphys-documentation.readthedocs.io/)
